templates/docs/patterns/logo-section/index.md

```diff
@@ -62,6 +62,8 @@ To import just this component into your project, copy the snippet below and incl
 @include vf-base;
 
 @include vf-p-logo-section;
+@include vf-p-strip;
```

Comment:
To use component in your project you don't necesarily need those. We just use them in an example.

Reply:
@bartaz revised!